src/App.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <div id="app">
+    <AppHeader />
+    <div class='thing'>
+      <router-view></router-view>
+    </div>
+  </div>
+</template>
+
+<script>
+import AppHeader from './components/AppHeader'
+
+export default {
+  name: 'app',
+  data () {
+    return {
+      msg: 'Hello, this is your computer speaking.',
```

Comment:
you can kill this object if you don't have any default data you want to include, or if you don't like it's props you can just make it an empty object.